src/tests/countrySlice.test.js

```diff
@@ -0,0 +1,40 @@
+import { formatCountries } from '../redux/country/countrySlice';
```

Comment:
- [ ] Great job so far however according to the project requirements, you could also perform a mocking data fetching and Redux Store connection. please address this. 👍🏻 

![image](https://github.com/TayyabKM/React-Capstone/assets/79058364/c1d99159-bba1-46ff-9074-337ed89946e1)